polyfills/Array/from/tests.js

```diff
@@ -7,7 +7,7 @@ it('has correct instance', function () {
 
 it('has correct name', function () {
 	function nameOf(fn) {
-		return Function.prototype.toString.call(fn).match(/function\s*([^\s]*)\(/)[1];
+		return Function.prototype.toString.call(fn).match(/function\s*([^\s]*)\s*\(/)[1];
```

Comment:
These helper functions were not fully correct as they did not take into consideration the fact that there can be any number of spaces between the function name and the opening parenthesis.